integration_tests/e2e/induction/updateSkills.cy.ts

```diff
@@ -0,0 +1,103 @@
+import Page from '../../pages/page'
```

Comment:
Sorry to be pedantic, but I think it would be better if we refer to "personal skills", rather than just "skills". Or at least I don't think we can call "personal interests" just "interests" as that would be confusing (given we have "in-prison" interests and "work" interests). Also, whilst unlikely, I guess they could add an "in-prison" skills in future?!

Reply:
I was planning on calling the personal interests tests `updatePersonalInterests.cy.ts`

I guess I'm thinking of the naming based around the screen, and (probably more so) the URL. The URL for this page  (skills) is `/prisoners/:prisonNumber/induction/skills`

But I can change this cypress test name to `updatePersonalSkills.cy.ts` if you like; but I'm not keen on changing the route, controller, validator, view etc at this point (they all use the simpler term "skills")

Reply:
Oh man, the inconsistency with just `skills` and `personal interests` is going to eat me alive! 🤣 

It's fine as it is then. At least it's called skills throughout (well, except where you interact with the API!).